app/models/concerns/assign_to_current_semester.rb

```diff
@@ -6,6 +6,7 @@ module AssignToCurrentSemester
   end
 
   def assign_to_current_semester
-    self.semester = Semester.current_semester unless semester.present?
+    # TODO: Settings are not initilized yet here
+    # self.semester = Semester.find(Settings.new.current_semester_id)
```

Comment:
Maybe we can do something like

    JUNE = 6
    today = Date.today
    year = today.year
    season = today.month < JUNE ? "Spring" : "Fall"
    Semester.create! year: year, season: season

To create a semester with the current year + season?